src/mantle/assets/class-asset.php

```diff
@@ -86,14 +86,21 @@ class Asset {
 	 *
 	 * @var bool
 	 */
-	protected $frontend = true;
+	protected bool $frontend = true;
 
 	/**
 	 * Enqueue in the admin area.
 	 *
 	 * @var bool
 	 */
-	protected $admin = true;
+	protected bool $admin = false;
```

Comment:
This changes previous behavior where assets were included front and back by default. If that's intended, it should be noted somewhere.

Reply:
Reverted!